Presentation.Web/Controllers/API/V1/UserController.cs

```diff
@@ -13,10 +13,14 @@
 using Core.DomainServices;
 using Core.DomainServices.Extensions;
 using Core.DomainServices.Generic;
+using Core.DomainServices.Queries;
+using Core.DomainServices.Queries.User;
 using Newtonsoft.Json.Linq;
+using Presentation.Web.Extensions;
 using Presentation.Web.Infrastructure.Attributes;
 using Presentation.Web.Models.API.V1;
 using Presentation.Web.Models.API.V1.Users;
+using Presentation.Web.Models.API.V2.Request.Generic.Queries;
```

Comment:
Add a bounded query model to the v1 model scope in stead of using the v2 model (so we can let them have their own life)